code/go/0chain.net/smartcontract/dbs/event/authorizer.go

```diff
@@ -128,3 +126,7 @@ func (edb *EventDb) updateAuthorizersTotalUnStakes(authorizer []Authorizer) erro
 func mergeUpdateAuthorizerTotalUnStakesEvents() *eventsMergerImpl[Authorizer] {
 	return newEventsMerger[Authorizer](TagUpdateAuthorizerTotalUnStake, withUniqueEventOverwrite())
 }
+
+func mergeAuthorizerHealthCheckEvents() *eventsMergerImpl[Authorizer] {
+	return (*eventsMergerImpl[Authorizer])(newEventsMerger[Authorizer](TagAuthorizerHealthCheck))
```

Comment:
This event merger would not be able to handle the events that has same index, that's why the `withUniqueEventOverwrite()` be used in most of the cases. For example the code used above:
```go
func mergeUpdateAuthorizerTotalUnStakesEvents() *eventsMergerImpl[Authorizer] {
	return newEventsMerger[Authorizer](TagUpdateAuthorizerTotalUnStake, withUniqueEventOverwrite())
}
```

Btw, we don't need to do the type conversion, so just be `newEventsMerger[Authorizer](TagAuthorizerHealthCheck, withUniqueEventOverwrite())`. Please also fix all the other merge events below in this PR.